engine/server/engine/main.go

```diff
@@ -261,6 +261,12 @@ func runMain() error {
 		}
 	}()
 
+	if serverArgs.RestartAPIContainers {
```

Comment:
moving the logic up, before starting the REST API because it will try to reconnect to all the APIC containers and we have to restart them before.